### PR TITLE
101527364-associate-space-with-space-quota-definition

### PR DESCRIPTION
**Pull request by o-orand**
This PR implements [Associate Space with the Space Quota Definition](http://apidocs.cloudfoundry.org/214/space_quota_definitions/associate_space_with_the_space_quota_definition.html)
[#101527364]

### CODE REVIEW

File: cloudfoundry-client-spring/src/test/java/org/cloudfoundry/client/spring/v2/spacequotadefinitions/SpringSpaceQuotaDefinitionsTest.java [hunk 1]
```diff
@@ -144,4 +198,5 @@ protected ListSpaceQuotaDefinitionsRequest getValidRequest() {
 
     }
 
+
```
Comment (nebhale) [inline cloudfoundry-client-spring/src/test/java/org/cloudfoundry/client/spring/v2/spacequotadefinitions/SpringSpaceQuotaDefinitionsTest.java]: Remove the unneeded extra line.

File: cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/spacequotadefinitions/AssociateSpaceQuotaDefinitionResponse.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import org.cloudfoundry.client.v2.Resource;
+
+/**
+ * The response payload for the Associate Space Group with the Space Quota Definition operation
+ */
+
```
Comment (nebhale) [inline cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/spacequotadefinitions/AssociateSpaceQuotaDefinitionResponse.java]: Remove unneeded extra line

**Comment by nebhale** [comment]
Great submission @o-orand.  I went ahead and made the minor cosmetic changes that I commented on (just wanted to have them called out so you could see them) before rebasing and merging in.  Please take a look at the commit message in 6b2fba3 for a template to use next time.

**Comment by o-orand** [comment]
Thanks @nebhale, I've tried to avoid extra line but it seems I've missed some. Next time ;-)

**Pull request closed without merging**